README.md

```diff
@@ -175,6 +180,11 @@ Sets the base of the current branch's PR to its upstream (parent) branch, as see
 git machete github retarget-pr
 ```
 
+Synchronize with the remote repository: delete untracked and unmanaged branches and checkout your open PRs: <br/>
```

Comment:
Yeah, probably including both `sync` and `checkout-prs` is too much

Reply:
And for someone who just starts using git-machete (as this is the primary audience of README, as opposed to readthedocs's potentially more advanced audience), some aspects of `clean`/`github sync` might be surprising, esp. the removal of unmanaged branches (well, they might have NO managed branches yet)

Reply:
You are right, i will remove `git machete clean` and leave only `git machete github sync`